### DISCUSSION TITLE
Linking to non-md file

### DISCUSSION THREAD

**Discussion by octvs** [artempyanykh/marksman#200] [Q&A] [answered]
Hey there, 
I'm trying to replace vimwiki. I only used navigation and it turns out I don't need vimwiki for it. I checked waikiki and mkdnflow but I intend to use helix instead of nvim so I'm up to not using a plugin for this purpose.

Navigation between markdown files seems to work flawlessly. Although I can't get to make links to non-md files work. I can understand this was never considered as a feature that would fall under the umbrella of marksman, but I still wanted to ask whether it is possible.

As an example I wanna follow up to code file as here
```
[ex1](src/ex1.cpp)
```
Cheers

**Comment by artempyanykh** [comment, marked as answer]
Hi @octaskin! Thanks for asking. Actually, I do want to support links to arbitrary files. Just haven't gotten the time to think it through. E.g. how to merge the existing completion for md-files with completion for non-md-files while maintaining a reasonable UX.
